packages/x-date-pickers/src/internals/hooks/useField/useField.ts

```diff
@@ -423,7 +430,6 @@ export const useField = <
     onChange: handleInputChange,
     onKeyDown: handleInputKeyDown,
     onMouseUp: handleInputMouseUp,
-    error: inputError,
```

Comment:
That's probably from my diff, but moving the `error` is useless here, we can keep it at the end of the spread